derived/derivation.adoc

```diff
@@ -90,6 +90,11 @@ IDerived "0..*" -> "1..*"  Node: base
 Each derived node references one or more base nodes.
 Any (base) node can have none or more derived nodes.
 
+[horizontal,labelwidth=12]
+QUESTION:: Should a derived node also be able to refer to features inside a node? Or do we leave that for specific classes implementing `IDerived`? 
```

Comment:
In the example below, the `PropertyViolation` points to a specific feature.

Assume the following concepts:
```
concept Person [id=P, key=pers]
  property firstName: string [id=PF, key=firstN]
  property lastName: string [id=PL, key=lastN]

concept EmptyNameViolation implements IDerived, IViolation [id=ENV, key=emptyNameV]
  property message: string [id=ENVM, key=msg]
  reference property: Property [id=ENVP, key=prop]
```

Original model contains instance:
```
Person A [id=2]
  firstName = "John"
  lastName = ""
```

Validation derived model contains instance:
```
EmptyNameViolation B [id=5]
  message = "lastName is empty!"
  base = --> A[id=2]
  property = --> lastName[id=PL]
```